main.css

```diff
@@ -1,3 +1,31 @@
-h1 {
-    color: red;
+body {
+    font-size: 18px;
+    font-family: "Avenir Next", "Avenir", -apple-system, system-ui,
+    -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+html, 
+body {
+    height: 100%;
+}
+
+.main-content {
+    display: flex;
+    height: 50%;
+}
+
+.text-editor {
+    flex: 1;
+    padding:  30px ;
+    border: 0;
+    resize: none;
+    background: #ffffff;
+    color: #333333;
+}
+
+.preview {
+    flex: 1;
+    padding: 0 50px;
+    background: #f3f3f3;
+    color: #3d3d3d;
 }
```

Comment:
In each file, make sure you have an extra line at the bottom.

Reply:
Noted.